content/post/kubestash-v2024.3.16/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Introducing KubeStash v2024.3.16
+date: "2024-03-19"
+weight: 10
+authors:
+- Md Anisur Rahman
+tags:
+- backup
+- cli
+- disaster-recovery
+- kubernetes
+- kubestash
+- restore
+---
+
+We are pleased to announce the release of `KubeStash v2024.3.16`, packed with new features and important bug fixes. You can check out the full changelog [HERE](https://github.com/kubestash/CHANGELOG/blob/master/releases/v2024.3.16/README.md). In this post, we'll highlight the key updates.
+
+### New Features
+
+Previously, some restore options such as VolumeSnapshots and volumeClaimTemplates were restricted in their functionality. We were not able to restore them to a namespace other than restoreSession. This was due to the requirement of creating a RestoreSession in the same namespace as the dataSource.
+
+To resolve this issue, we've introduced a new field called `.spec.dataSource.namespace` within the RestoreSession (CRD)([#104](https://github.com/kubestash/apimachinery/pull/104)). This field allows you to specify the namespace where the dataSource (i.e., repositories and snapshots) is located.
+
+So, with the addition of the `.spec.dataSource.namespace` field, users can now create a RestoreSession in a different namespace while referencing the namespace of the dataSource.
+
+> Note, if the `.spec.dataSource.namespace` field is left empty during restoration, the KubeStash operator will consider the namespace of the dataSource as equivalent to the namespace of the RestoreSession.
```

Comment:
the KubeStash operator will consider the DataSource is in the same namespace of the `RestoreSession`.